server/src/instant/superadmin/routes.clj

```diff
@@ -0,0 +1,60 @@
+(ns instant.superadmin.routes
+  (:require [compojure.core :refer [defroutes POST GET DELETE] :as compojure]
+            [ring.util.http-response :as response]
+            [clojure.string :as string]
+            [instant.util.uuid :as uuid-util]
+            [instant.model.app :as app-model]
+            [instant.model.instant-user :as instant-user-model]
+            [instant.util.exception :as ex]
+            [instant.util.http :as http-util])
+
+  (:import
+   (java.util UUID)))
+
+(defn req->auth-user! [req]
```

Comment:
Nit. I wonder if we should make shared module under `/auth` since this is now duplicated in `/dash/routes`